### PR TITLE
HAI-2402 Show car traffic nuisance categories in nuisance control plan page

### PR DESCRIPTION
**Pull request by umeetiusbaar**
# Description

Show car traffic nuisance categories in nuisance control plan page

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Other

# Instructions for testing

Open a project, draw some geometries for it and give nuisance values and go to "Haittojen hallinta" page. There should be car traffic nuisance with an index and indexed subcatagories under it.

# Checklist:

- [x] I have written new tests (if applicable)
- [x] I have ran the tests myself (if applicable)
- [ ] I have made necessary changes to the documentation, link to confluence
      or other location:

# Other relevant info

Please describe here if there is e.g. some requirements for this change or
other info that the tester/user needs to know.

### CODE REVIEW

**Comment by markohaarni** [comment]
According to Figma the sub sections should not have "Haittaindeksi" text above the index.

Screenshot 2024-06-24 at 10 15 53

File: src/domain/common/haittaIndexes/HaittaSubSection.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  oddEven?: 'odd' | 'even';
+  testId?: string;
+}>) {
+  return (
+    <Grid
+      templateColumns="1fr 24px"
+      gap="var(--spacing-xs)"
+      paddingX="var(--spacing-s)"
+      paddingY="var(--spacing-xs)"
+      background={oddEven === 'odd' ? 'var(--color-black-5)' : 'var(--color-black-10)'}
```
Comment (markohaarni) [inline src/domain/common/haittaIndexes/HaittaSubSection.tsx:22]: Might be better to do the background color with css pseudo-classes. In that way there would be no need for `oddEven` prop. For example:
```
<Grid
  templateColumns="1fr 24px"
  gap="var(--spacing-xs)"
  paddingX="var(--spacing-s)"
  paddingY="var(--spacing-xs)"
  _odd={{ backgroundColor: 'var(--color-black-5)' }}
  _even={{ backgroundColor: 'var(--color-black-10)' }}
>
```

**Comment by umeetiusbaar** [comment]
> According to Figma the sub sections should not have "Haittaindeksi" text above the index.
> 
> Screenshot 2024-06-24 at 10 15 53

That's right! Thank you for noticing. I set the `showLabel` parameter to `false`.

File: src/domain/common/haittaIndexes/HaittaSubSection.tsx (hunk 1 above)
Comment (umeetiusbaar) [reply src/domain/common/haittaIndexes/HaittaSubSection.tsx]: I did not know about this, thanks! Fixed as you suggested.

**Review by markohaarni — APPROVED** [review]

**Pull request merged**